middleware/src/main/java/br/ufrn/dimap/middleware/infrastructure/lifecycleManagement/modelo/IStaticRegistry.java

```diff
@@ -0,0 +1,6 @@
+package br.ufrn.dimap.middleware.infrastructure.lifecycleManagement.modelo;
+
+public interface IStaticRegistry

```

Comment:
Change interface name to StaticRegistry, following the pattern used by the others classes and interfaces